.github/workflows/linux_intel_oneAPI.yml

```diff
@@ -60,8 +60,10 @@ jobs:
           use-mamba: true
 
       - name: Update Conda Environment
+        shell: bash -l {0}
         run: |
-          mamba env update -n scipy-dev -f environment.yml
+          conda activate scipy-dev
+          conda install -c conda-forge pkg-config meson meson-python ninja numpy cython pybind11 pytest pytest-xdist pytest-timeout pytest-fail-slow pooch rich-click click doit pydevtool hypothesis
```

Comment:
`openblas` and `libblas=*=*openblas` are not listed here because we are using MKL installed in the previous step (from Intel website).